cmd/lifecycle/restorer.go

```diff
@@ -281,3 +290,16 @@ func (r *restoreCmd) restore(layerMetadata files.LayersMetadata, group buildpack
 	}
 	return nil
 }
+
+func (r *restoreCmd) getInsecureRegistryOptions(imageRef string) []remote.ImageOption {
```

Comment:
yes, this could be a temporary solution but I'm still in favour of fixing the `options` struct visibility in the imgutil library in order to be able to use the `InsecureHandler` everywhere we need it in lifecycle.
I'll do it but I'll add a @TODO or a //FIXME as a reminder :)